yaml/gateway-dep.yml

```diff
@@ -13,7 +13,7 @@ spec:
       serviceAccountName: faas-controller
       containers:
       - name: gateway
-        image: openfaas/gateway:0.9.14
+        image: openfaas/gateway:0.10.2
```

Comment:
Let's only change one thing at a time please? We are either updating the versions or we're splitting out the YAML files.

Reply:
Resolved